packages/components/src/local/progress-indicator/README.md

```diff
@@ -0,0 +1,28 @@
+# Progress Indicator
+
```

Comment:
I guess you've got some more content to go in here. I guess it's having a least one, so we're familiar with your intentions for language/tone/level of detail.

Reply:
Actually no, not really. The parts inside <!-- comment --> are auto-populated from inside storybook. I might add some extra comments to make that clearer though.

I'd say that this part:

`The Progress Indicator component is used to indicate the users progress through a form.`

Could be more fleshed out I guess but I wasn't sure what else to put.